FirebaseDev.podspec

```diff
@@ -16,6 +16,7 @@ Simplify your iOS development, grow your user base, and monetize more effectivel
   s.source           = { :git => 'https://github.com/firebase/firebase-ios-sdk.git', :tag => s.version.to_s }
   s.social_media_url = 'https://twitter.com/Firebase'
   s.ios.deployment_target = '8.0'
+  s.osx.deployment_target = '10.10'
```

Comment:
Mm, right right. On that note, curious if it might make sense to fill in FirebaseDev with subspecs/dependencies for the missing static components (RemoteConfig, AdMob, etc.)? Then FirebaseDev could actually be used in place of Firebase, existing on trunk distinctly until the final call on static/dynamic build/distribution is made (and until more of the subspecs are open sourced, I imagine).